api/v1beta1/common_types.go

```diff
@@ -128,6 +128,14 @@ type WatcherSpecCore struct {
 	// +kubebuilder:validation:Minimum=10
 	// APITimeout for Route and Apache
 	APITimeout *int `json:"apiTimeout"`
+
+	// +kubebuilder:validation:Optional
+	// NotificationsBusInstance is the name of the RabbitMqCluster CR to select
+	// the Message Bus Service instance used by the Watcher serviceto publish notifications.
+	// If undefined, the value will be inherited from OpenStackControlPlane.
+	// An empty value "" leaves the notification drivers unconfigured and emitting no notifications at all.
+	// Avoid colocating it with RabbitMqClusterName or other message bus instances used for RPC.
```

Comment:
I'm not sure I understand this last sentence. Does it mean that we should not reuse the same rabbitmq instance from `RabbitMQClusterName`?